include/bitcoin/explorer/commands/qrcode.hpp

```diff
@@ -175,7 +207,7 @@ class BCX_API qrcode
         )
         (
             "filename",
-            value<std::string>(&argument_.filename)->required(),
+            value<std::string>(&argument_.filename),
```

Comment:
Making this argument optional makes the preceding option optional as well, it's not clear to me that this is intended.